SilMock/tests/Google/Service/Directory/Resource/MembersTest.php

```diff
@@ -36,6 +36,24 @@ public function testInsert()
         self::assertTrue($addedMember instanceof GoogleDirectory_Member);
     }
 
+    public function testGet()
+    {
+        $groupEmailAddress = 'sample_group@groups.example.com';
+        $emailAddress = 'test@example.com';
+        $mockGoogleServiceDirectory = new GoogleMock_Directory('anyclient', $this->dataFile);
```

Comment:
It seems like this test assumes (but doesn't cause/verify) that the given email address is a member of the given group. Do you think it's worth adding code (now or later) to ensure that is the case as part of the "setup" part of this test (i.e. before the call to "members->get()")?

Reply:
This is why the test order is important currently, which is not perfection, but right now it needs to work.